shoop/admin/static_src/base/js/content-blocks.js

```diff
@@ -21,4 +21,9 @@ $(function() {
             $(this).find(".block-title").addClass("mobile-error-indicator");
         }
     });
+    // Activate first sidebar-list-item with errors
+    const $firstSidebarListItemWithErrors = $("a.sidebar-list-item.errors").first();
+    if ($firstSidebarListItemWithErrors) {
```

Comment:
No need for this `if` altogether, I meant (`$(selector)` never returns a falsy value).,

Just

```javascript
$("a.sidebar-list-item.errors").first().trigger("click");
```

should be plenty.